### PR TITLE
[Tile] Update content + Images for experimental tile

### PR DESCRIPTION
**Pull request by thyhmdo**
Will resolve https://github.com/carbon-design-system/carbon-website/issues/3261, https://github.com/carbon-design-system/carbon-website/issues/2943

- [x] Update the content 
- [x] Update the image

### CODE REVIEW

File: src/pages/components/tile/usage.mdx [hunk 1]
```diff
@@ -61,13 +63,25 @@ ecosystem below:
...
 
+### When not to use
+
+Tiles reside in the same plane as the background layer—they do not have
+elevation. Tiles organize essential information and have the same visual
+hierarchy as content within the same page.
+
+Do not add drop shadow to tiles and use them on screen to reveal secondary
+information, actions, or notifications. Use [modals](/components/modal/usage),
+[popover](/components/popover/usage), and [diaglog](/components/dialog/usage)
```
Comment (francinelucca) [inline src/pages/components/tile/usage.mdx:82]:
```suggestion
[popover](/components/popover/usage), and [dialog](/components/dialog/usage)
```

Comment (francinelucca) [reply src/pages/components/tile/usage.mdx:82]: [/components/dialog/usage](https://carbondesignsystem.com/components/dialog/usage) doesn't seem to be a link, I'm getting 404

File: src/pages/components/tile/usage.mdx [hunk 2]
```diff
@@ -387,6 +407,11 @@ are written out as a formula of width to height, such as 3:2. It’s important t
 remember that while two images can have the same aspect ratio, they can have
 varying images sizes.
 
+Experimental tiles are now available. This addition enhances accessibility and
+only applies to the visual, not the functions of tiles. Though we are not
+deprecating the current tiles, we encourage all design teams to use the new
+experimental tiles in their products moving forward.
+
```
Comment (francinelucca) [inline src/pages/components/tile/usage.mdx:414]: is ti correct for this text to be in an InlineNotification up top and here as well? 
image
image

File: src/pages/components/tile/usage.mdx [hunk 3]
```diff
@@ -399,13 +424,14 @@ varying images sizes.
...
 ### Alignment
 
-Always strive for left alignment whenver possible. Type in tiles should always
-be left aligned. Icons or icon buttons can be aligned to either left or right
-edge depending on the tiles’ complexity.
+#### Icon and call-to-action button
+
+With clicktable tiles, place the icon at the bottom right to indicate
+navigation. Use the interactive token for the icon. Avoid using interactive link
+that can be confusing as multiple click targets.
```
Comment (francinelucca) [inline src/pages/components/tile/usage.mdx:431]:
```suggestion
With clicktable tiles, place the icon at the bottom right to indicate
navigation. Use the interactive token for the icon. Avoid using interactive links
that can be confusing as multiple click targets.
```

File: src/pages/components/tile/usage.mdx [hunk 4]
```diff
@@ -399,13 +424,14 @@ varying images sizes.
...
 
-For example: In situations where there are both an icon or pictogram with an
-icon button, place the icon or pictorgram in the lower left corner of the tile
-and move the icon button (action) to the right corner.
+With non-clickable tiles(or base tile), place the interactive components, such
+as button or link, at the bottom left to indicate call-to-action.
```
Comment (francinelucca) [inline src/pages/components/tile/usage.mdx:434]: should there be other images under this section? it's calling for either bottom right or bottom left alignment but the pictures are showing top right
image

**Review by francinelucca — CHANGES_REQUESTED** [review]
just some questions and change suggestions

File: src/pages/components/tile/usage.mdx [hunk 5]
```diff
@@ -61,13 +71,25 @@ ecosystem below:
...
+### When not to use
+
+Tiles reside in the same plane as the background layer—they do not have
+**elevation**. Tiles organize essential information and have the same visual
+hierarchy as content within the same page.
+
+Do not add a drop shadow to tiles and use them on the screen to reveal secondary
+information, actions, or notifications. Use [modals](/components/modal/usage),
+[popover](/components/popover/usage), and [dialog](/components/dialog/usage)
+which have elevation and are appropriate for this function instead.
```
Comment (francinelucca) [inline src/pages/components/tile/usage.mdx:91]: we do not currently have a component page for dialog in website so this dialog link is returning 404: 
image

**Review by francinelucca — CHANGES_REQUESTED** [review]
Looks good! just one minor 404 issue.

Note: the first image wasn't loading for me in tile usage in the deployment but I ran this locally and it does load, it's probably just a cache issue that we shouldn't see in prod :) 
image

File: src/pages/components/tile/usage.mdx [hunk 6]
```diff
@@ -38,10 +40,18 @@ the issues.
...
-tiles have no pre-set styles for the content within them, the tile component
-usage guidance is purposefully high-level. It focuses specifically on the tile
-itself, not the structure of the information or interactive elements that the
-tile contains—that type of information will be found in the card pattern.
+tiles have no pre-set styles for the content inside, the tile component usage
+guidance focuses on basic tiles' functions. It does not include the complex
+structure of the information or additional interactive elements that a tile
+contains.
+
+<Row>
```
Comment (laurenmrice) [inline src/pages/components/tile/usage.mdx:48]: I am not sure why this image is not showing up. Maybe @francinelucca can help.

File: src/pages/components/tile/usage.mdx [hunk 7]
```diff
@@ -366,88 +400,95 @@ the chevron icon.
...
-remember that while two images can have the same aspect ratio, they can have
-varying images sizes.
+### Alignment
 
-<Row>
-<Column colLg={8}>
+#### Icon and call-to-action button
 
-![Example image of expandable tiles.](images/tile-usage-7.png)
+On clickable tiles, use the arrow icon with the `interactive` token in light
```
Comment (laurenmrice) [inline src/pages/components/tile/usage.mdx:431]: For the arrow icon, I think we decided on still using the `interactive` color token for all themes but just adjusting the color value of the token in the dark themes to be white (#ffffff). @jeanservaas to confirm.

File: src/pages/components/tile/usage.mdx [hunk 8]
```diff
@@ -457,7 +498,7 @@ hierarchical importance like navigation or catalog tiles.
 <Row>
 <Column colLg={12}>
 
-![Example image of how tiles are built on the narrow grid.](images/tile-usage-14.png)
+![Example image of how tiles are built on the narrow grid.](images/tile-usage-16.png)
```
Comment (laurenmrice) [inline src/pages/components/tile/usage.mdx:501]: There are some typos in this image.

Screen Shot 2023-02-14 at 9 10 30 AM

File: src/pages/components/tile/usage.mdx [hunk 9]
```diff
@@ -487,91 +528,34 @@ version.
...
-elevation. Do not add drop shadows to tiles. Tiles organize essential
-information and have the same visual hierarchy as content within the same page.
-Modals, popovers, and dialogs do have elevation but are not inherently part of
-the user's main workstream and are invoked on screen to reveal secondary
-information, actions, or notifications.
+### States
 
-### Call-to-actions
+For detailed visual information about the various states for this component, see
+the [Style](http://localhost:8000/components/tile/style) tab.
```
Comment (laurenmrice) [inline src/pages/components/tile/usage.mdx:551]:
```suggestion
the [Style](https://carbondesignsystem.com/components/tile/style) tab.
```

File: src/pages/components/tile/usage.mdx [hunk 10]
```diff
@@ -487,91 +528,34 @@ version.
...
-
-![Do not use link or highlight the icon color for clickable tiles.](images/tile-usage-23.png)
-
-  </DoDont>
-</DoDontRow>
-
-## Behaviors
+| State      | When to use                                                                                                                                         |
+| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Enabled_  | When a tile is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component    |
```
Comment (laurenmrice) [inline src/pages/components/tile/usage.mdx:555]: The image under this table should say `Multi-select tile` for the last tile.

Screen Shot 2023-02-14 at 9 18 15 AM

File: src/pages/components/tile/usage.mdx [hunk 11]
```diff
@@ -644,21 +625,27 @@ To learn more about how to build tiles correctly on the grid, see Carbon’s
 
 #### Aspect ratio
```
Comment (laurenmrice) [inline src/pages/components/tile/usage.mdx:626]: The `Grid` section above this needs a period at the end of the sentence.

File: src/pages/components/tile/usage.mdx [hunk 12]
```diff
@@ -644,21 +625,27 @@ To learn more about how to build tiles correctly on the grid, see Carbon’s
...
 
 #### Link
 
 Link has variants depending on the context. For further guidance, see Carbon’s
-[link](/components/buttons/usage).
+[link](https://carbondesignsystem.com/components/link/usage/).
+
+#### Spacing scale
+
+Link has variants depending on the context. For further guidance, see Carbon’s
```
Comment (laurenmrice) [inline src/pages/components/tile/usage.mdx:647]: This sentence seems to be a copy/paste from the `Link` section above it.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looking really great Thy! Just added a couple of comments. ⭐️

File: src/pages/components/tile/usage.mdx (hunk 6 above)
Comment (francinelucca) [reply src/pages/components/tile/usage.mdx:48]: @tay1orjones can you run the vercel deployment with a clean cache ? I don't think I have access

File: src/pages/components/tile/usage.mdx [hunk 13]
```diff
@@ -276,44 +298,51 @@ modules. Here are some common use cases for when to use tiles:
...
-variety of different use cases when building more complicating card features.
+variety of different use cases when building more complicated card features.
 
 | Variant                   | Purpose                                                                                                  |
 | ------------------------- | -------------------------------------------------------------------------------------------------------- |
 | [Base](#base)             | For high-level, short, and digestible content pieces such as features, plans, or services offered        |
 | [Clickable](#clickable)   | For prompting an action, navigating or directing to other pieces of information about the subject matter |
 | [Selectable](#selectable) | For presenting options to a user in a structured manner, such as a set of pricing plans                  |
-| [Expandable](#expandable) | For hiding and revealing a large amount of content in order to focus on specific pieces of info          |
+| [Expandable](#expandable) | For hiding and revealing a large amount of content to focus on specific pieces of info                   |
```
[the remark this answers is not in the document]
Comment (thyhmdo) [reply src/pages/components/tile/usage.mdx:317]: Jeannie suggested not to leave periods after these sentences that are not complete or full. But I'll check other tables if they're full sentences.

File: src/pages/components/tile/usage.mdx (hunk 6 above)
Comment (francinelucca) [reply src/pages/components/tile/usage.mdx:48]: can you see it now @laurenmrice ? https://carbondesignsystem-git-fork-thyhmdo-757abb-carbon-design-system.vercel.app/components/tile/usage/

File: src/pages/components/tile/usage.mdx [hunk 14]
```diff
@@ -276,44 +299,51 @@ modules. Here are some common use cases for when to use tiles:
...
+| [Expandable](#expandable) | For hiding and revealing a large amount of content to focus on specific pieces of information            |
 
 ### Base
 
 Base tiles are used to display information to the user, such as features or
 services offered. Base tiles are often seen on marketing pages to promote
-content or on dashboards that are highly interactive. These tiles can have
-internal calls-to-action (CTAs), such as a [button](/components/button/usage) or
-a [link](/components/link/usage).
+content or on highly interactive dashboards. These tiles can have internal
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:324]: highly-interactive

File: src/pages/components/tile/usage.mdx (hunk 7 above)
Comment (laurenmrice) [reply src/pages/components/tile/usage.mdx:432]: Update ^ we are making a new color token called `$icon-interactive`. We are tracking this in a separate issue.

File: src/pages/components/tile/usage.mdx [hunk 15]
```diff
@@ -276,44 +299,51 @@ modules. Here are some common use cases for when to use tiles:
...
-![Example image of base tiles.](images/tile-usage-1.png)
+![Example image of base tiles](images/tile-usage-2.png)
 
 </Column>
 </Row>
 
 ### Clickable
 
-Clickable tiles can be used as navigational items, where the entire tile is a
+Clickable tiles can be used as navigational items, where the entire tile is in a
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:338]: Clickable tiles can be used as navigational elements that redirect the user to a new page. In these situations, the entire tile is in a clickable state. Due to accessibility concerns, clickable tiles cannot contain separate internal CTAs but can contain pictograms, icons, or media such as illustrations or images.

File: src/pages/components/tile/usage.mdx (hunk 6 above)
Comment (laurenmrice) [reply src/pages/components/tile/usage.mdx:48]: Yes! Thank you for fixing it!

File: src/pages/components/tile/usage.mdx [hunk 16]
```diff
@@ -366,88 +401,95 @@ the chevron icon.
...
 <Column colLg={8}>
 
-![Tile anatomy](images/tile-usage-6.png)
+![Tile anatomy](images/tile-usage-8.png)
 
 </Column>
 </Row>
 
-1. **Container**: Includes a title, optional label, and the close icon.
+1. **Image (optional)** : The image follows the aspect ratios with the default
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:409]: The image size follows standard Carbon aspect ratios (in this case, with a default size of 4:3)

File: src/pages/components/tile/usage.mdx [hunk 17]
```diff
@@ -366,88 +401,95 @@ the chevron icon.
...
+
+2. **Container**: Contains the content area
 
-2. **Content area**: Contains the information and/or controls needed to complete
+3. **Content area**: Contains the information and/or controls needed to complete
    the modal's task. It can include message text and components.
 
 ### Sizing
 
 The **width** varies depending on three basic grid modes: wide, narrow, and
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:419]: Tile width varies depending on three basic grid modes: wide, narrow, and condensed. Tile height varies depending on the amount of content placed within it, but applies the [spacing scale](https://carbondesignsystem.com/guidelines/spacing/overview/#spacing-scale) and follows Carbon’s standard [aspect ratio](https://www.carbondesignsystem.com/guidelines/2x-grid/overview/#aspect-ratio)s. The minimum height starts with 2:1 aspect ratio. As the content grows, apply appropriate vertical spacers between sections within the content area.

File: src/pages/components/tile/usage.mdx [hunk 18]
```diff
@@ -366,88 +401,95 @@ the chevron icon.
...
+### Alignment
 
-<Row>
-<Column colLg={8}>
+#### Icon and call-to-action button
 
-![Example image of expandable tiles.](images/tile-usage-7.png)
+On clickable tiles, use the arrow icon with the `interactive` token in light
+themes and the `icon-primary` token for dark themes. Place the icon at the
+bottom right to indicate navigation. Avoid using interactive links or buttons
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:434]: Avoid confusing the user with multiple click targets (like interactive links or buttons) on clickable tiles.

File: src/pages/components/tile/usage.mdx [hunk 19]
```diff
@@ -366,88 +401,95 @@ the chevron icon.
...
+  <DoDont type="do" caption="Do right align the icon when it is by itself">
 
-</Column>
-</Row>
+![Example of "do" on clickable tile](images/tile-usage-9.png)
 
-### Alignment
+  </DoDont>
+
+  <DoDont type="do" caption="Do right align the icon when there is an additional text">
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:444]: Do right align the icon when there is additional text.

File: src/pages/components/tile/usage.mdx [hunk 20]
```diff
@@ -366,88 +401,95 @@ the chevron icon.
...
 
-![Example image of expandable tiles.](images/tile-usage-7.png)
+On clickable tiles, use the arrow icon with the `interactive` token in light
+themes and the `icon-primary` token for dark themes. Place the icon at the
+bottom right to indicate navigation. Avoid using interactive links or buttons
+that can be confusing as multiple click targets on clickable tiles.
 
-<Caption>Tiles get wider as the browser gets larger </Caption>
+<DoDontRow>
+  <DoDont type="do" caption="Do right align the icon when it is by itself">
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:438]: Do right align the icon when it is by itself.

File: src/pages/components/tile/usage.mdx [hunk 21]
```diff
@@ -366,88 +401,95 @@ the chevron icon.
...
+![Example of "do" on base tile](images/tile-usage-14.png)
 
   </DoDont>
 </DoDontRow>
 
-### Placement
-
 #### Tile groups
 
 Tile groups are a useful way of aligning tiles that have a strong relationship.
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:485]: Tile groups are useful for aligning tiles that have a strong relationship. Tile groups flow horizontally from left to right usually and have similar hierarchical importance as with navigation or catalog tiles.

File: src/pages/components/tile/usage.mdx [hunk 22]
```diff
@@ -482,96 +524,39 @@ version.
 
 - Standard layout tiles are the same in height and width as the rest of them in
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:525]: There are three basic layouts for tiles: standard, vertical masonry, and horizontal masonry. The standard layout will be the most commonly used version.

* 		Standard layout tiles are the same in height and width as all other tiles in the group.
* 		In a vertical masonry layout, tiles can vary in height, but are consistent in width.
* 		In a horizontal masonry layout, tiles can vary in width; different rows of tiles may vary in height, but tiles within a row should be consistent in height.

File: src/pages/components/tile/usage.mdx [hunk 23]
```diff
@@ -482,96 +524,39 @@ version.
...
-For **clickable tiles**, don’t highlight text or icons to avoid them being
-confused as links or clickable.
+## Universal behaviors
 
-<DoDontRow>
-
-  <DoDont type="do" caption="Do use text or icon for clickable tiles if needed.">
-
-![Do use text or icon for clickable tiles if needed.](images/tile-usage-22.png)
+The behaviors listed in this section are universal across all variants. For
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:545]: The behaviors listed in this section are universal across all tile variants. For behaviors that are unique to each variant, see each of the component variants sections below.

File: src/pages/components/tile/usage.mdx [hunk 24]
```diff
@@ -587,7 +572,7 @@ base tiles.
 
 <Caption>
   Clickable, expandable, and selectable tiles are clickable anywhere within the
-  tile.
+  tile
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:575]: tile. 

[Always put periods at the end of complete sentences. Sentence fragments don't get a periods.]

File: src/pages/components/tile/usage.mdx [hunk 25]
```diff
@@ -598,28 +583,25 @@ base tiles.
...
+CTAs if they are given their click targets and the tile’s click target is
+reduced to only the icon button.
 
 <Row>
 <Column colLg={8}>
 
 ![Clickable, expandable, and selectable tiles are clickable anywhere within the tile.](images/tile-usage-26.png)
 
+<Caption>
+  Expandable tiles can either be triggered as a whole or the click target is
```
Comment (jeanservaas) [inline src/pages/components/tile/usage.mdx:601]: Expandable tiles can either be triggered as a whole or contain internal CTAs only if they are given their own click targets. In these cases, the tile’s click target is reduced to only the icon button.

**Review by jeanservaas — CHANGES_REQUESTED** [review]
The only other thing has to do with these images that I talked to you about in Slack.

image

image





This is a weird design in that usually there are not buttons in cards with this aspect ratio. Also, we tend to use tertiary buttons on tiles that have buttons. Maybe for these two examples you can use a square tile. The button doesn't always span the distance of the card either. You found a couple of examples in product that looked better. 

Unfortunately, if you change the aspect ration of the button tiles, you'll probably want to change the aspect ratio of the tiles with links to match.

**Comment by thyhmdo** [comment]
@francinelucca everything looks good, except this image. The provided image has blue arrow icons, but the Preview site shows them in gray color. Can you please check on this? Thank you! 


image

**Comment by francinelucca** [comment]
> @francinelucca everything looks good, except this image. The provided image has blue arrow icons, but the Preview site shows them in gray color. Can you please check on this? Thank you!
> 
> image

@thyhmdo I wasn't able to get the arrow colors to show up on my local env either. After consulting on this with @tay1orjones , our theory is the image compression is causing it to lose the arrow colors. Unfortunately this being the case there's not much we can do 🥲

**Comment by thyhmdo** [comment]
> > @francinelucca everything looks good, except this image. The provided image has blue arrow icons, but the Preview site shows them in gray color. Can you please check on this? Thank you!
> > image
> 
> @thyhmdo I wasn't able to get the arrow colors to show up on my local env either. After consulting on this with @tay1orjones , our theory is the image compression is causing it to lose the arrow colors. Unfortunately this being the case there's not much we can do 🥲

Thank you for checking on this!

File: src/pages/components/tile/usage.mdx (hunk 13 above)
[the remark this answers is not in the document]
Comment (laurenmrice) [reply src/pages/components/tile/usage.mdx]: Okay, all the designers should follow this pattern for future documentation, so it is consistent. 👍

**Review by jeanservaas — COMMENTED** [review]
Images look good, just a couple of my text changes didn't make it in:

## 1

image

[Experimental tiles](https://react.carbondesignsystem.com/?path=/story/experimental-improved-contrast-tile--clickable) are now available. This addition enhances accessibility and only applies to the **visual appearance,** not the function of tiles. Though we are not deprecating the current tiles, we encourage all design teams to use the new experimental tiles in their products moving forward.


## 2


image

Tiles are reusable components that provide shortcuts to building cards and other modules. **Use tiles to group related information in flexible containers.** Here are some common use cases for when to use tiles:

## 3

image

Do right align the icon when there **is additional text**

## 4

All captions that are complete sentences need periods.

image

image

image

## 5
image

_Take out the caption here (it says the same thing as the paragraph above. Change paragraph above to read:_


Expandable tiles can either be triggered as a whole or contain internal CTAs only **if they are given their own click targets. In these cases, the tile’s click target is reduced to only the icon button.**

File: src/pages/components/tile/style.mdx [hunk 26]
```diff
@@ -11,19 +11,104 @@ import { white } from '@carbon/themes';
...
+The tile border applies to clickable, selectable and expandable tiles — not to
+base tiles.
+
+</InlineNotification>
+
+| Element          | Property         | Color token       |
+| ---------------- | ---------------- | ----------------- |
+| Tile: background | background-color | `$layer` \*       |
+| Tile border      | border           | `$border-tile` \* |
+| Arrow icon       | svg              | `$interactive`    |
```
Comment (laurenmrice) [inline src/pages/components/tile/style.mdx:25]:
```suggestion
| Arrow icon       | svg              | `$icon-interactive`    |
```

Comment (laurenmrice) [inline src/pages/components/tile/style.mdx:25]:
```suggestion
| Arrow icon       | svg              | `$icon-interactive`    |
```

File: src/pages/components/tile/style.mdx [hunk 27]
```diff
@@ -33,11 +118,21 @@ import { white } from '@carbon/themes';
 |         | min-width         | 128 / 8  | –             |
 |         | padding (minimum) | 16 / 1   | `$spacing-05` |
 
-<div className="image--fixed">
+<Row>
+<Column colLg={12}>
+
+![Structure and measurements for tiles](images/tile-style-11.png)
```
Comment (laurenmrice) [inline src/pages/components/tile/style.mdx:124]: This image has a typo. Should say `modes`
Screen Shot 2023-03-01 at 1 24 25 PM

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks good Thy! Just a couple of things on the Style tab.

**Review by jeanservaas — APPROVED** [review]

**Review by laurenmrice — APPROVED** [review]

**Review by francinelucca — CHANGES_REQUESTED** [review]
LGTM! one anchor link detail and we should be good

**Review by francinelucca — APPROVED** [review]

**Pull request merged**